### PR TITLE
Revise SDL-0184 Dismiss Alert

### PR DESCRIPTION
**Pull request by joeljfischer**
Revise SDL-0184 with changes:

1. Change name of proposal and new RPC to Cancel Interaction
2. Add parameters to `CancelInteraction`
3. Add new parameters to cancelable RPCs

### CODE REVIEW

**Comment by joeljfischer** [comment]
@theresalech this is ready, thanks!

File: proposals/0184-dismiss-alert.md [hunk 1]
```diff
@@ -7,35 +7,128 @@
...
+    </param>
 </function>
+```
 
-<function name="DismissAlert" functionID="DismissAlertID" messagetype="response">
+Additional Changes:
+
+```xml
+<function name="PerformInteraction" functionID="PerformInteractionID" messagetype="request">
+    <description>Triggers an interaction (e.g. "Permit GPS?" - Yes, no, Always Allow).</description>
```
Comment (theresalech) [inline proposals/0184-dismiss-alert.md]: Should "no" be capitalized here?

File: proposals/0184-dismiss-alert.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Dismiss Alert RPC
+# Cancel Interaction RPC
 
 * Proposal: [SDL-0184](0184-dismiss-alert.md)
```
Comment (theresalech) [inline proposals/0184-dismiss-alert.md]: Do you want to change this file name to cancel-interaction?

File: proposals/0184-dismiss-alert.md (hunk 1 above)
Comment (joeljfischer) [reply proposals/0184-dismiss-alert.md]: That's the exact description from `PerformInteraction` in the RPC spec. I just copied and pasted.

File: proposals/0184-dismiss-alert.md (hunk 2 above)
Comment (joeljfischer) [reply proposals/0184-dismiss-alert.md]: Fixed

**Comment by joeljfischer** [comment]
@theresalech Please re-review when you get a chance. Thanks!

File: proposals/0184-cancel-interaction.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
+# Cancel Interaction RPC
+
+* Proposal: [SDL-0184](0184-dismiss-alert.md)
```
Comment (theresalech) [inline proposals/0184-cancel-interaction.md]: Need to rename proposal name here.

Comment (joeljfischer) [reply proposals/0184-cancel-interaction.md]: 😅 Fixed

**Pull request merged**